### PR TITLE
Anna branch 01

### PR DESCRIPTION
**Pull request by avsank**

### CODE REVIEW

**Review by Olesea1986 — APPROVED** [review]
Let's see!!!

**Review by asuman75 — APPROVED** [review]
hello

**Review by IrinaMakaryan — APPROVED** [review]
Well done

File: configuration.properties [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
 userName2=User54
 userName3=User84
 userName4=User114
 password=Userpass123
 browser=chrome
 environment=http://qa.trycloud.net/index.php/login?clear=1
+invalidPassword=yais2021
+#invalid credentials
+
+
```
Comment (altyshaibraim) [inline configuration.properties:12]: good job

**Review by altyshaibraim — COMMENTED** [review]
good work

**Pull request merged**